### ISSUE TITLE
Merge cells range is not deleted within cells

### ISSUE THREAD

**Issue by AlexPravdin** [PHPOffice/PhpSpreadsheet#867]
This is:

```
- [X] a bug report
```

### Test script
```php
<?php
$workbook = new \PhpOffice\PhpSpreadsheet\Spreadsheet();
$sheet = $workbook->getActiveSheet();
$sheet->mergeCells('A1:B2');
$writer = \PhpOffice\PhpSpreadsheet\IOFactory::createWriter($workbook, "Xlsx");
$writer->save('1.xlsx');

$workbook = \PhpOffice\PhpSpreadsheet\IOFactory::load('1.xlsx');
$sheet = $workbook->getActiveSheet();
$sheet->removeRow(1, 2);
$writer = \PhpOffice\PhpSpreadsheet\IOFactory::createWriter($workbook, "Xlsx");
$writer->save('2.xlsx');

$workbook = \PhpOffice\PhpSpreadsheet\IOFactory::load('2.xlsx');
print_r($workbook->getActiveSheet()->getMergeCells());
```

### What is the expected behavior?
Empty array output because we have deleted cells which were merged.

### What is the current behavior?
Script output:
```
Array
(
    [A1:B2] => A1:B2
)
```
Despite merged cells were deleted merge range remain exists.

PhpSpreadsheet: 1.6.0
PHP: 7.3.1

**Comment by AlexPravdin** [comment]
Is it a way of how issues are solved? Just ignore them and bot will close them?

**Comment by oleibman** [comment]
Fixed quite recently by PR #4465. Leaving issue closed, removing stale label.
